website/content/en/preview/provisioner.md

```diff
@@ -22,6 +22,11 @@ spec:
   # If omitted, the feature is disabled, nodes will never scale down due to low utilization
   ttlSecondsAfterEmpty: 30
 
+  # If omitted, Karpenter will create nodes for pods in any namespace
+  namespaceSelector:
```

Comment:
Can we make document an example where you can pick namespaces by name? Doing so with labels is nice, but perhaps not the common case. I think you can do this with match fields.

Reply:
The namespace selector only does match labels.  Pod affinity terms have both a namespace selector and a list of namespaces:

```
	// namespaces specifies a static list of namespace names that the term applies to.
	// The term is applied to the union of the namespaces listed in this field
	// and the ones selected by namespaceSelector.
	// null or empty namespaces list and null namespaceSelector means "this pod's namespace"
	// +optional
	Namespaces []string `json:"namespaces,omitempty" protobuf:"bytes,2,rep,name=namespaces"`

	// A label query over the set of namespaces that the term applies to.
	// The term is applied to the union of the namespaces selected by this field
	// and the ones listed in the namespaces field.
	// null selector and null or empty namespaces list means "this pod's namespace".
	// An empty selector ({}) matches all namespaces.
	// This field is alpha-level and is only honored when PodAffinityNamespaceSelector feature is enabled.
	// +optional
	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty" protobuf:"bytes,4,opt,name=namespaceSelector"

```

Maybe we should do the same, or just drop namespace selector and only allow a list of namespaces?

Reply:
Added a namespace list to match pod affinity terms.